Author: Sven Joachim <svenjoac@gmx.de>
Description: Correct kf1-kf4 in mlterm* terminfo entries
 Cherry-picked fix from the 20201205 upstream patchlevel:
  - Correct mlterm3 kf1-kf4
  - Add flash to mlterm3
Bug-Debian: https://bugs.debian.org/975322
Forwarded: not-needed
Last-Update: 2021-01-01

---
 misc/terminfo.src |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/misc/terminfo.src
+++ b/misc/terminfo.src
@@ -5984,15 +5984,15 @@ mlterm|multi lingual terminal emulator,
 # with ncurses, tack and vttest -TD
 mlterm3|multi lingual terminal emulator,
 	bce, AX,
-	blink=\E[5m, invis=\E[8m, is2=\E[!p\E[?3;4l\E>,
-	kf1=\E[11~, kf2=\E[12~, kf3=\E[13~, kf4=\E[14~,
-	rs2=\E[!p\E[?3;4l\E>,
+	blink=\E[5m, flash=\E[?5h$<100/>\E[?5l, invis=\E[8m,
+	is2=\E[!p\E[?3;4l\E>, rs2=\E[!p\E[?3;4l\E>,
 	sgr=\E[0%?%p6%t;1%;%?%p2%t;4%;%?%p4%t;5%;%?%p1%p3%|%t;7%;%?
 	    %p7%t;8%;m%?%p9%t\E(0%e\E(B%;,
 	use=ansi+enq, use=ansi+rep, use=ecma+italics,
 	use=ecma+strikeout, use=xterm+app, use=xterm+pcf2,
 	use=xterm+pcc2, use=xterm+pce2, use=xterm+meta,
-	use=xterm+alt+title, use=xterm+sm+1006, use=mlterm2,
+	use=xterm+alt+title, use=xterm+sm+1006,
+	use=vt100+pfkeys, use=mlterm2,
 
 # This is mlterm 2.9.3's mlterm.ti, with some additions/corrections -TD
 #
